lrpar/src/lib/astar.rs

```diff
@@ -62,6 +62,9 @@ where
         }
 
         if !neighbours(true, &n, &mut next) {
```

Comment:
Does this always mean "no neighbours were found within our `RECOVERY_TIME_BUDGET`"? Could this accidentally panic in the case that the search was exhausted (but within the allotted time) ?

Reply:
At the moment it's exclusively used to mean "timeout".

Reply:
Fair enough